### ISSUE TITLE
Hypenation of titles

### ISSUE THREAD

**Issue by cgogolin** [cgogolin/quantum-journal#44]
The following was sent to me from Pieter Kok:

The title is typeset very large (using `\Huge`), and because of it words almost unavoidably get hyphenated. I think this is undesirable behaviour for the title. 

A simple modification that solves both problems is to redefine the command `\@printtitle`. I have replaced `\Huge` with `\huge` and inserted a `\raggedright` command to prevent hyphenation:
```
\def\@printtitle{%
  {\huge\href{http://quantum-journal.org/?s=\detokenize\expandafter{\@title}}{%
      \begin{minipage}{1.0\linewidth}%
        \raggedright%
        \color{quantumviolet}{\@title}%
      \end{minipage}}%
  }%

```

**Comment by cgogolin** [comment]
Well, I think the `\Huge` titles were a design decision and I am reluctant to change this (at least for short titles), but hyphenation in titles is definitely not great. I already received some negative feedback from other sources about the justified titles so switching to `\raggedright` is probably a good idea.
A more sophisticated fix for this would be to reduce the text size if the title is longer than one line? Would that be a useful feature?

**Comment by pieterkok** [comment]
I would be in favour of dropping from \Huge to \huge when the title goes over two lines, although I can see funny behaviour for titles that span two lines in \Huge, but only one line in \huge. In that case I'd say use \huge.

**Comment by cgogolin** [comment]
Yes, that's exactly the behavior I had in mind. Check if the title fits in one line with size `\Huge` otherwise use `\huge`.

**Comment by dbranford** [comment]
`\raggedright` to prevent hyphenation makes sense, but would personally prefer consistency with size, given a number of the existing titles already go over a line (and that can only increase with `\raggedright`) probably `\huge` makes more sense to me.  I'd expect having two different articles which are supposed to be styled the same but with different sized headers to just look strange.

**Comment by cgogolin** [comment]
I checked back with our professional designers and since authors can anyway already chose between single and double column as well as different font sizes for the whole document a variable size for the title of either \Huge or \huge was seen as the best option in the end. This, together with the `\raggedright` macro as suggested above has been implemented with aef0b2f.
If the title does not fit on line line it size `\Huge` it is printed in size `\huge`. Together with `\raggedright` this should avoid hyphenation in nearly all real life titles. Thanks for reporting!

**Comment by pieterkok** [comment]
Hi Christian,

Thanks for the feedback!

Pieter.


*Dr. Pieter Kok (EU immigrant)*Reader in Quantum Information Theory
Head of First Year Physics
The University of Sheffield
Hicks building, Hounsfield Road
S3 7RH, Sheffield, UK
http://www.pieter-kok.staff.shef.ac.uk


On 6 December 2017 at 16:49:39, Christian Gogolin (notifications@github.com)
wrote:

I checked back with our professional designers and since authors can anyway
already chose between single and double column as well as different font
sizes for the whole document a variable size for the title of either \Huge
or \huge was seen as the best option in the end. This, together with the
\raggedright macro as suggested above has been implemented with aef0b2f
<https://github.com/quantum-journal/quantum-journal/commit/aef0b2f1fcc4de698ff5523142ec7d765b75c18c>
.
If the title does not fit on line line it size \Huge it is printed in size
\huge. Together with \raggedright this should avoid hyphenation in nearly
all real life titles. Thanks for reporting!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/quantum-journal/quantum-journal/issues/44#issuecomment-349701409>,
or mute the thread
